deploy/charts/envs/mixer_private.yaml

```diff
@@ -0,0 +1,122 @@
+# Helm config
+mixer:
+  gcpProjectID: datcom-mixer-statvar
+  serviceName: mixer.endpoints.datcom-mixer-statvar.cloud.goog
+
+  useTMCFCSVData: true
+  tmcfCSVBucket: datcom-public
+  tmcfCSVFolder: food
+
+memdbJSON: |
```

Comment:
If the file gets larger, is there a way to load it from a json file as before?

Reply:
yes, for example, the mcfs can be set like the following when calling helm.

--set-file mixer.configmaps."encode.mcf"=deploy/mapping/encode.mcf

Value of mixer.configmaps.encode.mcf is set from deploy/mapping/encode.mcf

Thinking out loud, I think for large files, we should consider loading from gcs instead.
(current deployment yaml  "--memdb_path=/datacommons/memdb")

Reply:
Got it. These files won't be too large, but maybe too long (hundreds of lines) to set inline in yaml